handlers/handlers.go

```diff
@@ -99,3 +121,84 @@ func HomepageRender(rend RenderClient, cli *codelist.Client) http.HandlerFunc {
 		return
 	}
 }
+
+//ListPageRender ...
+func ListPageRender(rend RenderClient, cli CodeListClient) http.HandlerFunc {
+	return func(w http.ResponseWriter, req *http.Request) {
+		ctx := req.Context()
+		vars := mux.Vars(req)
+		codeListID := vars["codeListID"]
+		logData := log.Data{
+			codeListID: codeListID,
+		}
+		var page list.Page
+
+		codeListEditions, err := cli.GetCodeListEditions(codeListID)
+		if err != nil {
+			log.ErrorCtx(ctx, errors.WithMessage(err, "error getting editions for a code-list"), logData)
+			setStatusCode(req, w, err)
+			return
+		}
+
+		if codeListEditions.Count > 0 {
+			edition := codeListEditions.Items[0]
+			page.Metadata.Title = edition.Label
+
+			log.InfoCtx(ctx, "getting codes for edition of a code list", log.Data{"edition": edition})
+			codesData, err := cli.GetCodes(codeListID, edition.Edition)
```

Comment:
could this var just be `codes` instead of `codeData`? not sure what value `Data` is adding here, esp as I can't see a collision with a different `codes` var